app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,58 @@
+class VendorsController < ApplicationController
+  def index
+    @all_vendors = Vendor.all.order(name: :asc)
```

Comment:
It's a great user experience for these to be ordered, good job sorting them here <3